protocols/grpc/agent.proto

```diff
@@ -270,6 +270,13 @@ message TtyWinResizeRequest {
 	uint32 column = 4;
 }
 
+message KernelModule {
+	// This field is the name of the kernel module.
+	string name = 1;
+	// This field are the options for the kernel module.
```

Comment:
Nit: might be worth mentioning in the comment that the options/parameters are whitespace-delimited key=value pairs which are passed verbatim to `modprobe(8)`.